Installation of leveldb fails

Hi,

We are trying to install the alastria-node on a Ubuntu 16.04 64bit machine with 2 cpu cores, 4gb of ram and 30gb of disk space.

We followed the instructions in the readme, executing the ```bootstrap.sh``` script. However, it looks like the leveldb installation fails:

```
...
Cloning into 'leveldb'...
remote: Counting objects: 1846, done.
remote: Total 1846 (delta 0), reused 0 (delta 0), pack-reused 1846
Receiving objects: 100% (1846/1846), 997.59 KiB | 0 bytes/s, done.
Resolving deltas: 100% (1274/1274), done.
Checking connectivity... done.
mkdir out-shared
mkdir out-shared/db
mkdir out-shared/port
mkdir out-shared/table
mkdir out-shared/util
mkdir -p out-shared/helpers/memenv
g++ -I. -I./include -std=c++0x -fno-builtin-memcmp -pthread -DOS_LINUX -DLEVELDB_PLATFORM_POSIX=1 -DLEVELDB_ATOMIC_PRESENT -DHAVE_CRC32C=0 -DHAVE_SNAPPY=0 -O2 -DNDEBUG -DLEVELDB_SHARED_LIBRARY -DLEVELDB_COMPILE_LIBRARY -fPIC -fvisibility=hidden -c db/builder.cc -o out-shared/db/builder.o
In file included from ./port/port.h:14:0,
                 from ./db/filename.h:14,
                 from db/builder.cc:7:
./port/port_posix.h:50:36: error: ‘LEVELDB_IS_BIG_ENDIAN’ was not declared in this scope
 static const bool kLittleEndian = !LEVELDB_IS_BIG_ENDIAN;
                                    ^
Makefile:425: recipe for target 'out-shared/db/builder.o' failed
make: *** [out-shared/db/builder.o] Error 1
```

--- a/scripts/bootstrap.sh
+++ b/scripts/bootstrap.sh
@@ -2,6 +2,11 @@
 
 set -e
 
+function installcmake {
+  wget "https://cmake.org/files/v3.10/cmake-3.10.3-Linux-x86_64.sh" -O /tmp/cmake-3.10.3-Linux-x86_64.sh
+  sudo sh /tmp/cmake-3.10.3-Linux-x86_64.sh --skip-license --prefix=/usr/local
+}
+
 DIR=`echo $PWD | xargs dirname | xargs dirname`
 OS=$(cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's\"\\g')
 if [ $OS = "centos" ] || [ $OS = "rhel" ];then
@@ -70,14 +75,12 @@ if [ $OS = "centos" ] || [ $OS = "rhel" ];then
   echo "Installing LEVELDB"
   git clone https://github.com/google/leveldb.git
   cd leveldb/
-  git checkout 0fa5a4f
+  installcmake
+  cmake .
   make
-  sudo scp -r out-static/lib* out-shared/lib* /usr/local/lib/
-  sudo cp /usr/local/lib/libleveldb.* /usr/lib64/
-  cd include/
-  sudo scp -r leveldb /usr/local/include/
+  make install
   sudo ldconfig || true
-  cd ../..
+  cd ..
   sudo rm -r leveldb
   cd ..
   echo "Cleaning CACHE RPM"
@@ -162,13 +165,12 @@ elif [ $OS = "ubuntu" ];then
   #LEVELDB FIX
   git clone https://github.com/google/leveldb.git
   cd leveldb/
-  git checkout 0fa5a4f
+  installcmake
+  cmake .
   make
-  sudo scp -r out-static/lib* out-shared/lib* /usr/local/lib/
-  cd include/
-  sudo scp -r leveldb /usr/local/include/
+  make install
   sudo ldconfig
-  cd ../..
+  cd ..
   rm -r leveldb
 
 #INSTALACION ETHEREUM

